data/src/languages_update.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import collections
+import json
+import operator
+import os
+import sys
+
+import unicodedataplus
+from data.src.codes import LANGUAGES_PATH, TSV_DIRECTORY_PATH
```

Comment:
newline between here. `Unicodedata is third-party`, `data` is part of this package.